.github/workflows/stale_branches.yml

```diff
@@ -0,0 +1,20 @@
+name: Run O11y Stale Branch POC
+
+on:
+  workflow_dispatch: {}
```

Comment:
Let's remove this and just let it be automatic. If we find we want to be able to trigger manually later on, we can add this back.

Reply:
This has been done so I'll resolve this conversation.